httpserver/http_connection.cpp

```diff
@@ -342,7 +345,7 @@ void connection_t::process_storage_test_req(uint64_t height,
                                             const std::string& tester_addr,
                                             const std::string& msg_hash) {
 
-    LOKI_LOG(trace, "Performing storage test, attempt: {}", repetition_count_);
+    LOKI_LOG(debug, "Performing storage test, attempt: {}", repetition_count_);
```

Comment:
I would like to keep this as trace, it will print over 1000 times per test where the message isn't present.
We already print the total attempt number in another log after we give up

Reply:
updated